asn1ate/sema.py

```diff
@@ -719,6 +721,19 @@ def __str__(self):
     __repr__ = __str__
 
 
+class Imports(SemaNode):
+    def __init__(self, elements):
+	self.module2symbols = { }
```

Comment:
In the X.680 spec, these are called 'SymbolsImported', so I suggest you follow that name: `symbols_imported`.